src/regmod/utils.py

```diff
@@ -63,7 +157,27 @@ def create_spline(self, vec: np.ndarray = None) -> XSpline:
                        include_first_basis=self.include_first_basis)
 
 
-def sizes_to_sclices(sizes: np.array) -> List[slice]:
+def sizes_to_sclices(sizes: List[int]) -> List[slice]:
```

Comment:
Typo in function name?

Reply:
Ahh, yes!! This typo started two years ago, it is amazing it still exists to this day XD. Just fixed it :)